Jellyfin.Api/Helpers/FileStreamResponseHelpers.cs

```diff
@@ -107,6 +126,7 @@ public static class FileStreamResponseHelpers
             // Headers only
             if (isHeadRequest)
             {
+                httpContext.Response.Headers[HeaderNames.ContentLength] = state.EstimateContentLength || isHeadRequest ? GetEstimatedContentLength(state) : string.Empty;
```

Comment:
This boolean expression seems a bit unnecessary 😉

Reply:
Cut / paste direct from 6.4 - want to see if this is the culprit before doing anything else.

Reply:
Seems to be https://emby.media/community/index.php?/topic/21181-estimate-content-length-when-transcoding/

Reply:
DLNA Spec
```
HTTP/1.0 HEAD request
Some implementations utilize the HEAD request to try to predetermine the amount of memory required to process a GET request. Some servers may not know that size of the content because it may be dynamic. In such cases, the responses will not contain a CONTENT-LENGTH header field. As such, control points shall not rely on the CONTENT-LENGTH header field being specified for a HEAD response.

HTTP/1.1 General
When a device or control point implements HTTP/1.1, all requirements of HTTP/1.0 shall be maintained, with the exception of the CONTENT-LENGTH header field, which shall not be specified when doing chunked transfers.
```

Reply:
Looks like Panasonic don't adhere to DLNA specs, as they rely on the content-length header being present.

Reply:
Are we sure that we're not just sending an incorrect value? I can imagine asp.net either sets it to 0 or sets it to chunked

Reply:
network sniff is provided in the bug report :- https://github.com/jellyfin/jellyfin/issues/4697

0 is being sent as the current code returns a zero.